dimod/reference/samplers/simulated_annealing.py

```diff
@@ -38,16 +39,18 @@ class SimulatedAnnealingSampler(Sampler):
     Examples:
         This example solves a two-variable Ising model.
 
-        >>> import dimod
-        ...
-        >>> response = dimod.SimulatedAnnealingSampler().sample_ising(
-        ...                  {'a': -0.5, 'b': 1.0}, {('a', 'b'): -1})
-        >>> response.record.energy      # doctest: +SKIP
-        array([-1.5, -1.5, -1.5, -1.5, -1.5, -1.5, -1.5, -1.5, -1.5, -1.5])
+        >>> h = {'a': -0.5, 'b': 1.0}
+        >>> J = {('a', 'b'): -1.5}
+        >>> sampleset = dimod.SimulatedAnnealingSampler().sample_ising(h, J)
 
     """
     properties = None
+
     parameters = None
+    """dict: Keyword arguments accepted by the sampling methods.
+
+    Contents are exactly `{'beta_range': [], num_reads': [], 'num_sweeps': []}`
```

Comment:
Will it be clear to our users that [] might not be a list?

Reply:
In this case it is always an empty list, that's what 'Contents are exactly' means